FLAnimatedImageDemo/RootViewController.m

```diff
@@ -59,6 +59,9 @@ - (void)viewWillAppear:(BOOL)animated
     NSData *data1 = [NSData dataWithContentsOfURL:url1];
     FLAnimatedImage *animatedImage1 = [FLAnimatedImage animatedImageWithGIFData:data1];
     self.imageView1.animatedImage = animatedImage1;
+    self.imageView1.loopCompletionBlock = ^(FLAnimatedImageView *imageView, NSUInteger loopLeft) {
+        NSLog(@"imageView1 - loopCompleted - %ld loop(s) left", (unsigned long)loopLeft);
```

Comment:
For most GIFs on the internet(tm) ;) you'll get "imageView1 - loopCompleted - 0 loop(s) left" because they loop forever. I'd like to avoid cluttering up the console log of the demo project with those frequent logs that don't carry much value.